packages/toolpad-app/src/toolpad/DomLoader.tsx

```diff
@@ -362,25 +366,54 @@ const UNDOABLE_ACTIONS = new Set<DomActionType>([
   'DESELECT_NODE',
 ]);
 
+export const getInitialPageDomView = (location: Location, defaultPageNodeId?: NodeId): DomView => {
```

Comment:
This looks like the inverse of the `getPathnameFromView`. Perhaps we could put the two functions physically together and name them similarly?

`getPathnameFromView` and `getViewFromPathname`?